rhel8/profiles/e8.profile

```diff
@@ -1,5 +1,10 @@
 documentation_complete: true
 
+metadata:
+    reference: https://www.cyber.gov.au/acsc/view-all-content/essential-eight
```

Comment:
Can we please change this to https://www.cyber.gov.au/acsc/view-all-content/publications/essential-eight-linux-environments